index.bs

```diff
@@ -959,14 +908,13 @@ string-valued keys. Values may be any type that has a valid encoding in JSON. It
 
     This structure is used by the client to compute the following quantities:
 
-    : <dfn dfn>clientDataJSON</dfn>
-    :: This is the <a>UTF-8 encoded</a> JSON serialization [[RFC7159]] of a {{ClientData}} dictionary. Any valid JSON
-        serialization may be used by the client. This specification imposes no canonicalization requirements. Instead, the
-        {{ScopedCredentialInfo}} and {{AuthenticationAssertion}} structures contain the actual serializations used by the client
-        to generate them.
+    : <dfn dfn>JSON-serialized client data</dfn>
```

Comment:
I see. Rewriting this as an algorithm is a great idea, and it would also help with the previous comment. Thanks for pointing this out.